radio/src/strhelpers.cpp

```diff
@@ -560,7 +560,7 @@ char *getSwitchPositionName(char *dest, swsrc_t idx)
   return dest;
 }
 
-char *getSourceString(char *dest, mixsrc_t idx)
+char *getSourceString(char* const dest, mixsrc_t idx)
```

Comment:
`const char*`

Reply:
That's impossible because `dest` is in the role of an input/output parameter.

As above, `const char* dest` and `char* const dest` are different things.

In this case the parameter-variable `dest` itself is read-only. This is what most variables (named objects) should be, because it prevents changing the semantic of an variable.

Reply:
Ok, then it does not have any added value. All the functions using `dest` here take `char*` as a param, so the change seems unnecessary to me.

Reply:
Refactored that also: now uses array-designator that can't be retargeted.

According to the`const`: my stragegiy is to `const`-all-thing. And here it is (better: was) really neccessary because this function returns `dest`. So it would be wrong to modify `dest` inside this function.
Making objects (local variables, parameters), `const` will prevent to give them a different _meaning_ when changing their value.